src/renderer/components/drawer/drawer-item.tsx

```diff
@@ -11,6 +12,7 @@ export interface DrawerItemProps extends React.HTMLAttributes<any> {
   renderBoolean?: boolean; // show "true" or "false" for all of the children elements are "typeof boolean"
 }
 
+@withTooltip
```

Comment:
Since we're using `<Badge/>` to show a tooltip, we don't need any changes in current file (`drawer-item`). Badge component itself already has all needed decorators.

Reply:
Thank you, I'll remove the decorator and `TooltipDecoratorProps`